### DISCUSSION TITLE
유니티로 개발한 건물 (산업 공장) 을 등록하면?

### DISCUSSION THREAD

**Discussion by Alkims001** [naverz/zepeto-studio-kor#669] [4. General]
유니티로 개발한 건물 (산업 공장) 을 등록하면 제페토 맵으로 사용 가능 한가요?

맵으로 등록시 개발된 형태에 변화가 생기나요?

![유니티로 제작된 가상공간](https://user-images.githubusercontent.com/104487551/165459228-0d25958e-8e23-4676-9486-903310c5299a.jpg)

아래 개발 환경 예시 입니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 
아래 가이드라인에 위배되지 않는 컨텐츠라면 본인이 개발한 건물을 비롯하여 
에셋 스토어를 이용한 어셋 모두 사용 가능합니다. 😊

https://github.com/naverz/zepeto-studio-kor/discussions/595
